news/2017-11-29-mapping-with-rewards-2-bounties-complete.md

```diff
@@ -8,22 +8,22 @@ Long overdue, but not forgotten. The second set of mapping bounties have finally
 
 ![](/wiki/shared/news/<post filename>/mwr-banner.jpg)
```

Comment:
`wiki/shared/news/banners/mwr-banner.jpg`

you put the banner there so filename for `<post filename>` syould reflect `2017-11-29-mapping-with-rewards-2-bounties-complete` as a folder and that's where the file should be

Reply:
applied